comms/src/outbound_message_service/outbound_message_service.rs

```diff
@@ -88,7 +88,10 @@ where DS: DataStore
             CommsPublicKey::shared_secret(&self.node_identity.secret_key, &dest_node_public_key).to_vec();
         let ecdh_shared_secret_bytes: [u8; 32] =
             ByteArray::from_bytes(&ecdh_shared_secret).map_err(|e| OutboundError::SharedSecretSerializationError(e))?;
-        Ok(chacha20::encode(message_envelope_body, &ecdh_shared_secret_bytes))
+        Ok(ChaCha20::seal_with_integral_nonce(
```

Comment:
Consider making use of a type alias such as CommsCipher that you can specify in "types.rs".

Reply:
Sure, done.